sdcflows/workflows/gre.py

```diff
@@ -0,0 +1,98 @@
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+"""
+
+.. _b0mapping:
```

Comment:
```suggestion
.. _gre-fieldmaps:
```

And search and replace this anchor if used elsewhere.